app/database_etl/test_evaluation_handler/test_eval_link_handler.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+
+TEST_EVAL_CSV_LOCATION = 'test_evaluation_handler/test_evaluations.csv'
+
+
+def get_isotype_list(isotype_str):
+    if pd.isna(isotype_str): return []
+    if isotype_str == "Total Ig": return ["IgM", "IgG", "IgA"]
```

Comment:
any reason why we're returning a set sometimes and a list other times?

Reply:
Changed to always return list